File: SimpleCalc.js
function Calculator(a,b,c)
{
  let result;
  switch(c)
    {
      case '+':
        result=a+b;
        break;
      case '-':
        result=a-b;
        break;
      case '*':
        result=a*b;
        break;
      case '/':
        if(b==0)
        {
          console.log("Error! Division by Zero is not Possible");
          break;
        }
        result=a/b;
        break;
      default:
        console.log("Operator is Invalid");
        return;
    }
  console.log("The result is: "+result);
}

function StartCalculator(){
  const prompt=require("prompt-sync")();

  console.log("Welcome to Simple Calculator");

  while(true)
    {
      const a=parseFloat(prompt("Enter the first number:"));
      const operator=prompt("Enter an operator:");
      const b=parseFloat(prompt("Enter the Second number:"));

      Calculator(a,b,operator);

      const continueResponse=prompt("Do you want to perform another Calculation(yes/no):");
      if(continueResponse.toLowerCase()=="no")
      {
        break;
      }

    }
  console.log("Thankyou for using the Calculator!!!");

}
StartCalculator();




    

File: SimpleCalc.java
import java.util.Scanner;

public class SimpleCalc{
  public static void main(String[] args)
  {
    Scanner sc=new Scanner(System.in);
    double a,b,result=0;
    char operator;
    

    while(true)
      {
        System.out.println("Enter the first number:");
        a=sc.nextDouble();

        System.out.println("Enter the Operator:");
        operator=sc.next().charAt(0);

        System.out.println("Enter the second number:");
        b=sc.nextDouble();

        switch(operator)
          {
            case '+':
              result=a+b;
              break;
            case '-':
              result=a-b;
              break;
            case '*':
              result=a*b;
              break;
            case '/':
              if(b==0)
              {
                System.out.println("Division by 0 is not possible");
                break;
              }
              result=a/b;
              break;
            default:
              System.out.println("Operator is Invalid");
              continue;
          }
      System.out.println("The Result is: "+result);
      System.out.print("Do you want to perform another calculation? (yes/no): ");
          String continueResponse = sc.next();

          if (continueResponse.equalsIgnoreCase("no")) {
              break;
          }
      }

      
      sc.close();
      System.out.println("Thank you for using the calculator!");
  }
}

              

    
